internal/diff/diff_test.go

```diff
@@ -108,14 +109,10 @@ func TestCompareEmptyDirectories(t *testing.T) {
 	dirOwnerInfo := fs.OwnerInfo{UserID: 1000, GroupID: 1000}
 	dirMode := os.FileMode(0o777)
 
-	cid, _ := index.IDFromHash("p", []byte("sdkjfn"))
-	dirObjectID1 := object.DirectObjectID(cid)
-
-	cid, _ = index.IDFromHash("i", []byte("dfjlgn"))
-	dirObjectID2 := object.DirectObjectID(cid)
-
-	dir1 := createTestDirectory("testDir1", dirModTime, dirOwnerInfo, dirMode, dirObjectID1)
-	dir2 := createTestDirectory("testDir2", dirModTime, dirOwnerInfo, dirMode, dirObjectID2)
+	oid1 := oidForString(t, "p", "sdkjfn")
+	oid2 := oidForString(t, "p", "dfjlgn")
```

Comment:
For consistency, use `k` since this is the prefix used for directories in `snapshotfs`.

```suggestion
	oid1 := oidForString(t, "k", "sdkjfn")
	oid2 := oidForString(t, "k", "dfjlgn")
```